src/Compilers/VisualBasic/Portable/PredefinedPreprocessorSymbols.vb

```diff
@@ -1,14 +1,11 @@
 ﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 Imports System.Collections.Immutable
-Imports Microsoft.CodeAnalysis.Text
-Imports Microsoft.CodeAnalysis.VisualBasic.Symbols
-Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
 
 Namespace Microsoft.CodeAnalysis.VisualBasic
     Public Module PredefinedPreprocessorSymbols
 
-        Friend Const CurrentVersionNumber = 15.3
+        Friend Const CurrentVersionNumber = 15.6
```

Comment:
> Friend Const CurrentVersionNumber = 15.6 [](start = 7, length = 41)

This looks very fragile, it is pretty easy to miss this change when a new version is added. I suggest we change this constant to a readonly shared field with initializer that calls MapSpecifiedToEffectiveVersion with latest, then calls GetErrorName on the result and parses its return as Double.